server/controllers/refer/referralsController.ts

```diff
@@ -245,12 +245,10 @@ export default class ReferralsController {
       TypeUtils.assertHasUser(req)
 
       if (req.body.confirmation !== 'true') {
-        req.flash('errors', [
-          {
-            href: '#confirmation',
-            text: 'Please confirm that the information you have provided is complete, accurate and up to date',
-          } as unknown as string,
-        ])
+        req.flash(
+          'confirmationError',
+          'Please confirm that the information you have provided is complete, accurate and up to date',
```

Comment:
Let's remove "Please" from here - [according to](https://design-system.service.gov.uk/components/error-message/#be-clear-and-concise) the GOV.UK design system it "implies the user has a choice" 😁

Reply:
"Confirm that" sounds a little harsh though - does it need further revision? "I confirm that"?

Reply:
I think it's ok to be direct, I don't think it's too harsh!